app/app.py

```diff
@@ -46,6 +46,11 @@ def _get_configured_parser():
                        default=False, action='store_true',
                        help='repeat the classification on the latest '
                        'user-defined set of drawings')
+
+    group2 = parser.add_mutually_exclusive_group()
```

Comment:
Shouldn't `--symbol` always occur with `--learning`? I think we have 3 exclusive sets of arguments:

* `-l` and `-sym`
* `-s`
* `-r` and `-sym`

So we should not allow the user to start the app with both `-s` and `-sym` at the same time.